code/citadel/organs/genitals.dm

```diff
@@ -32,6 +32,12 @@
 	update_appearance()
 	update_link()
 
+////Size-changing procs should return TRUE if size change happened, FALSE if not
+/obj/item/organ/genital/proc/grow_size()
+
+/obj/item/organ/genital/proc/shrink_size()
+
+
```

Comment:
These both should be combined into `AdjustSize(amt)` so it can accept an argument for changing the size. Look at the arousal setting procs for clues for achieving this. This proc would do `organ.size = (organ.size + amt)` and clamp it within the bounds set through the defines in `cit_defines.dm`. Doing this allows you to do increase and decreases in the same proc.

After that, add another proc called `SetSize(size)` which can be used to force a size change to a specific point.